.github/workflows/ci-data.yml

```diff
@@ -42,7 +42,7 @@ jobs:
 
       - uses: google-github-actions/auth@v2
         with:
-          credentials_json: ${{ secrets.GCP_CREDENTIALS }}
+          workload_identity_provider: "${{ secrets.WIF_PROVIDER_NAME }}"
```

Comment:
Per https://github.com/google-github-actions/auth/blob/main/README.md#workload-identity-federation-through-a-service-account do we want to add the currently used service account name here as a `service_account` key?

The alternative is that we use the workload identity pool as the principal in our IAM permissions, so we'd need to grant it permission to do all the same things the current service account can do.

Reply:
Let's wait a second on adding the service account. I think I can mail a change internally to grant the pool access to BigQuery resources needed to run the tests. Will try that first and see if additional approvals would be needed.

Reply:
Looks like I can grant these permissions directly, copying the same project-level permissions as the existing service account(s). Ignore this thread.